athena-dynamodb/src/test/java/com/amazonaws/athena/connectors/dynamodb/DynamoDBIT.java

```diff
@@ -0,0 +1,169 @@
+/*-
+ * #%L
+ * athena-dynamodb
+ * %%
+ * Copyright (C) 2019 - 2020 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.dynamodb;
+
+import com.amazonaws.athena.connector.integ.IntegrationTestBase;
+import com.amazonaws.services.athena.model.Row;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+import software.amazon.awscdk.core.Stack;
+import software.amazon.awscdk.services.iam.Effect;
+import software.amazon.awscdk.services.iam.PolicyDocument;
+import software.amazon.awscdk.services.iam.PolicyStatement;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Integration-tests for the DynamoDB connector using the Integration-test Suite.
+ */
+public class DynamoDBIT extends IntegrationTestBase {
```

Comment:
Please rename to be consistent with our class naming convention, e.g. DynamoIntegrationTest

Reply:
Changed to `DynamoDBIntegrationTest` (consistent with other test files, e.g: `DynamoDBRecordHandlerTest`